src/main/java/eci/ieti/leafOut/backEnd/service/impl/ParkServiceImpl.java

```diff
@@ -19,7 +20,9 @@
 
     @Override
     public List<Park> getAllParks() throws LeafOutPersistenceException {
-        return parkRepository.findAll();
+        List<Park> parks = new ArrayList<Park>();
+        parkRepository.findAll().forEach(park -> parks.add(park));
```

Comment:
This implementation is correct but anyway take a look into the methods of the newer java versions for creating immutable lists